languages/de.js

```diff
@@ -1,68 +1,63 @@
-(function(){
-  var juration;
-  if (typeof require !== 'undefined')
-    juration = require('juration');
-  else
-    juration = window.juration;
-  
-  if (juration && juration.UNITS){
-    juration.UNITS.seconds.patterns = ['Sekunde', 'Sekunden', 'sek', 's'];
-    juration.UNITS.minutes.patterns = ['Minute', 'Minuten', 'min', 'm(?!s)'];
-    juration.UNITS.hours.patterns = ['Stunde', 'Stunden', 'h'];
-    juration.UNITS.days.patterns = ['Tag', 'Tage', 'T', 't'];
-    juration.UNITS.weeks.patterns = ['Woche', 'Wochen', 'w'];
-    juration.UNITS.months.patterns = ['Monat', 'Monate', 'M', 'Mo', 'Mnt'];
-    juration.UNITS.years.patterns = ['Jahr', 'Jahre', 'J', 'Jr'];
-    
-    juration.UNITS.seconds.formats = {
-      'chrono': '',
-      'micro':  's',
-      'short':  'sek',
-      'long':   'Sekunde',
-      'plural': 'Sekunden'
+const setGermanUnits = UNITS => {
```

Comment:
Since you're making this much of a change, can you adapt this to import json files for each of the languages instead of having to maintain a javascript file for each?

Reply:
sounds good

Reply:
I will update to json in a different PR (depending on whether or not Smartling supports arrays)

Reply:
King and I discussed this and recognized that there wasn't a way to request an array of possible translations from Smartling.  I spoke with them today to give them details about the request and explain the need.  Steve Zapata at Smartling is going to circle around with his team and get back to us.  As a result, we will be postponing this change.